native/objcexport-header-generator/test/org/jetbrains/kotlin/backend/konan/tests/ObjCExportHeaderGeneratorTest.kt

```diff
@@ -48,6 +48,17 @@ class ObjCExportHeaderGeneratorTest(private val generator: HeaderGenerator) {
         doTest(headersTestDataDir.resolve("simpleEnumClass"))
     }
 
+    @Test
+    fun `test - enumClassWithNamedObjCEnum`() {
+        doTest(headersTestDataDir.resolve("enumClassWithNamedObjCEnum"))
+    }
+
+    @Test
+    @TodoAnalysisApi
```

Comment:
This ignores too much: not only the case not supported in AA, but also some supported cases.
It is reasonable to split the test to ignore only the absolutely necessary cases.